share/messages_to_clients.txt

```diff
@@ -9361,6 +9361,9 @@ ER_EXPLICIT_DECRYPTION_OF_ONLINE_ENCRYPTED_TABLESPACE
 ER_SEQUENCE_TABLE_SIZE_LIMIT
   eng "The number of records generated by SEQUENCE_TABLE() cannot exceed %llu (%llu requested). Try increasing @@tf_sequence_table_max_upper_bound to a larger value."
 
+ER_D_T_E_CHANGE_WHILE_ENCRYPTION_THREADS_NOT_ZERO
```

Comment:
Do we have test case that shows this error?

Reply:
I have extended the encryption.default_table_encryption_var test.